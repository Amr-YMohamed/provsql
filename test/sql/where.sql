\set ECHO none
SET search_path TO public,provsql;

SELECT provenance_eq('19257535-6aaf-5275-b02b-899c48576553',1,2);
SELECT * from provenance_circuit_extra;

