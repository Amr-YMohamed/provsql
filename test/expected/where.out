\set ECHO none
            provenance_eq             
--------------------------------------
 825d9471-e8a5-585e-bfb9-f11ad5c75484
(1 row)

                 gate                 | info | info_eq 
--------------------------------------+------+---------
 825d9471-e8a5-585e-bfb9-f11ad5c75484 |    1 |       2
(1 row)

